content/en/catalyst/fund9/challenges/grow-africa-grow-cardano/campaign-brief.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Grow Africa, Grow Cardano"
+linkTitle: "Grow Africa, Grow Cardano"
+type: docs
+hide_feedback: true
+description: "How do we continue to seed and grow Cardano in Africa in 2022?"
+---
+<img src="https://cardano.ideascale.com/community-library/accounts/93/936143/Public/10-Grow-Africa-Grow-Cardano-1daa76.png" style="width:500px;height500px">
+
+Fund Size: **500,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/396613) and [browse ideas]().
```

Comment:
Do we need to change the campaign link to this link, 'https://cardano.ideascale.com/c/campaigns/26600/about'?